app/Events/StatusUpdateEvent.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App\Events;
+
+use App\Models\Status;
+use Illuminate\Foundation\Events\Dispatchable;
+use Illuminate\Queue\SerializesModels;
+use Illuminate\Support\Facades\Log;
+
+class StatusUpdateEvent {
```

Comment:
This also needs to be dispatched from `\App\Http\Controllers\Frontend\Admin\StatusEditController::edit`